.buildscript/build_swift_docs.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright 2019 Square Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script uses Jazzy.
+# https://github.com/realm/jazzy
+# It requires ruby, bundler (2.x), CocoaPods, and Jazzy to build.
+# gem install bundler jazzy
+
+# Usage: build_swift_docs SHA REPO_URL_OR_PATH OUTPUT_DIR SPACE_SEPARATED_SCHEMES
+function build_swift_docs() {
+    SHA=$1
+    REPO="$2"
+    OUTPUT_DIR="$(realpath "$3")"
+    SCHEMES="$4"
+    ORIG_DIR="$(pwd)"
+    DIR=swiftdocs-clone
```

Comment:
`/tmp/swiftdocs-clone`?

Reply:
Easier to troubleshoot if something fails if you just stay in the same directory tree.